### ISSUE TITLE
Shared images sometimes stay 'stickied' at the bottom

### ISSUE THREAD

**Issue by vlur** [vector-im/element-android#2829]
**Describe the bug**
Sometimes, but not always, images shared to a room will stay 'stuck' at the bottom of the chat, forcing the sender to scroll up every time to see newer/more recent messages.

Deleting the images deletes them where they appear in the chat history, but the versions at the bottom of the sender's screen remain unchanged and still display.

Long press on one of the pictures brings up a dialog with information -- the sender's name and avatar, a long filename (e.g. '1234567890987654321.jpg'), the date the image was originally sent -- but no options to do anything except close the dialog.

Clearing cache and data has no effect.

**To Reproduce**
Can't reliably reproduce, but when it has happened before:
1. Long press/hold an image in web browser (Bromite, in my case).
2. 'Share image' to Element
3. Select destination room and wait for upload to finish

**Expected behavior**
Older messages should scroll off the screen once newer messages arrive.

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: ZTE Axon 7
 - OS: Resurrection Remix based on Nougat (Android 7.1)

**Additional context**
 - App version and store: 1.0.17 from GPlay
 - Homeserver: matrix.org

**Comment by benparsons** [comment]
This may be a manifestation of #516

**Comment by vlur** [comment]
Definitely seems to be the same issue. Sorry for the dupe. I did try to search the issues first before opening a new one, honest.

FYI: I managed to get rid of the stuck messages by going into Settings -> General -> Clear cache, which just worked. Interestingly, doing the same thing in the Android system app settings for Element did not. (On Android: Settings -> Apps -> Element -> Storage -> Clear cache)
